models/FirebaseQuery.go

```diff
@@ -24,9 +24,10 @@ type FirebaseQuery struct {
 	OrdDir firestore.Direction
 }
 
+// Initializes the query with values from strings (from header), sets Pg = 1 and OrdDir = Asc as default
 func NewFirebaseQuery(u string, p string, q string, o string, od string) FirebaseQuery {
```

Comment:
Golang wants us to write the name of the method/func before the comment it self. So it should be // NewFirebaseQuery ...